apps/web/src/scenes/ContentPages/PostsFeaturePage.tsx

```diff
@@ -24,58 +24,64 @@ export default function PostsFeaturePage({ pageContent }: Props) {
       <Head>
         <title>Gallery | Posts</title>
       </Head>
-      <StyledPage gap={96}>
-        <StyledContent align="center" gap={96}>
-          <StyledIntro gap={48} align="center">
-            <VStack gap={32}>
-              <VStack>
-                <StyledHeading>Introducing</StyledHeading>
-                <StyledHeading>
-                  <strong>Posts</strong>
-                </StyledHeading>
-              </VStack>
-              <StyledSubheading>{pageContent.introText}</StyledSubheading>
+      <PostsFeaturePageContent pageContent={pageContent} />
+    </>
+  );
+}
+
+export function PostsFeaturePageContent({ pageContent }: Props) {
```

Comment:
just moved the page content out into its own component so we can use it in the modal